### PR TITLE
onnxruntime 0.5.0 (new formula)

### PR DESCRIPTION
**Pull request by ankane**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/onnxruntime.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+class Onnxruntime < Formula
+  desc "Cross-platform, high performance scoring engine for ML models"
+  homepage "https://github.com/microsoft/onnxruntime"
+  url "https://github.com/microsoft/onnxruntime/releases/download/v0.5.0/onnxruntime-osx-x64-0.5.0.tgz"
```
Comment (SMillerDev) [inline Formula/onnxruntime.rb]: All formulae need to be build from source, unless the binaries are cross platform. See https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae

**Review by SMillerDev — CHANGES_REQUESTED** [review]

File: Formula/onnxruntime.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  homepage "https://github.com/microsoft/onnxruntime"
+  url "https://github.com/microsoft/onnxruntime.git",
+      :tag      => "v0.5.0",
+      :revision => "1f8019bc9208c5ddf3c59561570f654888369746"
+
+  depends_on "cmake" => :build
+  depends_on "python" => :build
+
+  def install
+    system "python3", "tools/ci_build/build.py", "--build_dir", "build/Linux",
```
Comment (SMillerDev) [inline Formula/onnxruntime.rb]: It might be better to refer to this as `#{bin}/python3`

Comment (SMillerDev) [inline Formula/onnxruntime.rb]: is `build/Linux` the correct directory for a macOS build?

File: Formula/onnxruntime.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+           "--config", "RelWithDebInfo", "--build_shared_lib", "--skip_submodule_sync"
```
Comment (SMillerDev) [inline Formula/onnxruntime.rb]: We probably don't want debug info in the formula

**Comment by SMillerDev** [comment]
I see upstream responded. We'll await their new version

**Comment by ankane** [comment]
Hey @SMillerDev, thanks for all the feedback. Unfortunately, it's a bit out of my expertise to get it running on Sierra. Unless there's a way to mark the formula as High Sierra or above, let's close and revisit once support for Sierra is dropped.

**Comment by ankane** [comment]
Hey @SMillerDev, it looks like Homebrew CI has added Catalina and dropped Sierra. Is it possible to reopen this (or should I create a new PR)?

**Pull request closed without merging**